packages/regenerator-runtime/runtime.js

```diff
@@ -709,3 +709,18 @@ var regeneratorRuntime = (function (exports) {
   // the regeneratorRuntime variable at the top of this file.
   typeof module === "object" ? module.exports : {}
 ));
+
+try {
+  regeneratorRuntime = runtime;
+} catch (accidentalStrictMode) {
+  // This module should not be running in strict mode, so the above
+  // assignment should always work unless something is misconfigured. Just
+  // in case runtime.js accidentally runs in strict mode, we can escape
+  // strict mode using a global Function call. This could conceivably fail
+  // if a Content Security Policy forbids using Function, but in that case
+  // the proper solution is to fix the accidental strict mode problem. If
+  // you've misconfigured your bundler to force strict mode and applied a
+  // CSP to forbid Function, and you're not willing to fix either of those
+  // problems, please detail your unique predicament in a GitHub issue.
```

Comment:
CSP errors are normal errors, right? So people which see it will follow the first entry in the stack trace and find this comment?

Reply:
That’s right, they’re runtime errors due to actually creating and calling a `Function`, not some sort of static error based on the appearance of `Function` in the code, as far as I know.

Reply:
I think should handle exception for this line code
```js
try {
  Function("r", "regeneratorRuntime = r")(runtime);
} catch (error) {
  console.warn('something')
}

```